src/components/Pages/Unit/widgets/UnitMaterials/Materials.js

```diff
@@ -65,7 +66,7 @@ const Materials = ({ unit = undefined, t, playlistComponent = null }) => {
     },
   ];
 
-  if (unit.content_type === CT_VIDEO_PROGRAM_CHAPTER || unit.content_type === CT_VIRTUAL_LESSON) {
+  if (unit.content_type === CT_VIDEO_PROGRAM_CHAPTER || unit.content_type === CT_VIRTUAL_LESSON || unit.content_type === CT_CLIP) {
```

Comment:
Please replace it with includes()
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes